src/front-end/src/App.js

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import { Layout, Menu } from 'antd';
 import './App.css';
 import { NavLink, Switch, Route } from 'react-router-dom';
-import MentorshipForm from './components/MentorshipForm';
+import MentorForm from './components/MentorForm';
 import Login from './components/Login';
-import MentorHome from './components/MentorHome';
```

Comment:
Can you add this line back in, as well as line 25 (surrounded by a Menu.item tag), and line 33? The MentorHome component is for representing a logged-in mentor

Reply:
KK I think I deleted it because it triggered prettier errors.

Reply:
Please make sure you keep the mentor home tab we had.